lib/record_store/record.rb

```diff
@@ -18,6 +18,12 @@ def initialize(record)
     end
 
     def self.build_from_yaml_definition(yaml_definition)
+      case yaml_definition.fetch(:type)
+      when 'ALIAS'
+        if yaml_definition.key?(:cname)
+          yaml_definition[:alias] = yaml_definition.delete(:cname)
+        end
+      end
```

Comment:
I think I like this for a safety precaution... we can remove it after shipping for all the existing zones